src/Views/BingoView.js

```diff
@@ -154,11 +155,43 @@ export const BingoView = ({ activities }) => {
     activities && setSquareCount(activities.length)
   }, [activities])
 
-
+  const COMPLETE_GAME = gql`
+  mutation modifyGame($id: Int!, $win: Boolean!, $activities: [String!]){
+    modifyGame(input: {params: {
+      id: $id
+      win: $win
+      activities: $activities
+      }
+     }
+    )
+    success
+  }
+  `
+  const [completeGame, { data, loading, error }] = useMutation(COMPLETE_GAME)
+    if (loading) console.log('Submitting...');
```

Comment:
Nice work to make this even better rather than using console.log show users feedback on the page.